test/per_file_coverage.sh

```diff
@@ -15,7 +15,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/memory:74.5" # tcmalloc code path is not enabled in coverage build, only gperf tcmalloc, see PR#32589
 "source/common/network:94.4" # Flaky, `activateFileEvents`, `startSecureTransport` and `ioctl`, listener_socket do not always report LCOV
 "source/common/network/dns_resolver:91.4"  # A few lines of MacOS code not tested in linux scripts. Tested in MacOS scripts
-"source/common/quic:93.4"
+"source/common/quic:93.3"
```

Comment:
What can we do to keep the coverage higher? Which lines are uncovered?

Reply:
I did not touch this quic file tbh.. not sure why it dropped a little bit

Reply:
Oh, wacky. Are you seeing this problem on tip of tree? If so, mind forking this change off into a stand-alone PR?

Reply:
Merge main and this can be removed.